paper/examples/flagstat/flagstat.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
Please add a license header and a package level comment explaining what flagstat.go does - one sentence is enough. The license header needs to have a blank line between it and the package comment so that it doesn't become part of the godoc.